server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -698,6 +699,27 @@ export class AgenticChatController implements ChatHandlers {
         return results
     }
 
+    #validateToolResult(toolUse: ToolUse, result: ToolResultContentBlock) {
+        let maxToolResponseSize
+        switch (toolUse.name) {
+            case 'fsRead':
+                maxToolResponseSize = 200_000
```

Comment:
nit: Should we store these as constants somewhere?

Reply:
I think we should but not sure where, maybe in runtimes?

Reply:
Possibly. I haven't really looked into that package to be honest. Maybe we can just add it to the constants file here or create one for agenticChat: https://github.com/aws/language-servers/blob/main/server/aws-lsp-codewhisperer/src/language-server/chat/constants.ts